torch/csrc/jit/codegen/cuda/lower_expr_sort.cpp

```diff
@@ -1398,6 +1398,9 @@ std::vector<Expr*> ExprSegmentationSorter::getExprs() const {
 
 std::vector<Expr*> reorderExprsForComputeAt() {
   auto fusion = FusionGuard::getCurFusion();
+  if (fusion->exprs().empty()) {
```

Comment:
so it's just a false alarm?!

I'm wondering how is the segmented graph different, when we have tensor with 1d with static size 0, vs 1d with a dynamic sized tensor.

Reply:
Yeah, it is a false alarm. And in the fusion, it was a 0d tensor, not a 1d 0 sized tensor.

Reply:
Ha, so it's a scalar tensor and that causes different segmentation? This is still pretty mysterious to me...

Note that I'm not holding anything against this PR..., we should merge it since Naoya already stamped.

Reply:
It is not segmented at all.